src/SDL/types.ts

```diff
@@ -1,3 +1,17 @@
-export type RWMode = "a" | "a+" | "r" | "r+" | "w" | "w+" | "ab" | "ab+" | "rb" | "rb+" | "wb" | "wb+";
+import { u16 } from "../types.ts";
+
+export type RWMode =
+  | "a"
+  | "a+"
+  | "r"
+  | "r+"
+  | "w"
+  | "w+"
+  | "ab"
+  | "ab+"
+  | "rb"
+  | "rb+"
+  | "wb"
+  | "wb+";
```

Comment:
TODO: Should probably just do away with this file and move RWMode into a more appropriate file name.